srcpkgs/noisetorch/template

```diff
@@ -0,0 +1,24 @@
+# Template file for 'noisetorch'
+pkgname=noisetorch
+version=0.11.3
+revision=1
+archs="x86_64*"
+wrksrc=NoiseTorch-${version}
+build_style=gnu-makefile
+hostmakedepends="go"
+depends="polkit"
+short_desc="Real-time microphone noise suppression on Linux"
+maintainer="Soc Virnyl S. Estela <renegan.ronin@gmail.com>"
+license="GPL-3.0-or-later"
+homepage="https://github.com/lawl/NoiseTorch"
+distfiles="https://github.com/lawl/NoiseTorch/archive/${version}.tar.gz"
```

Comment:
Since the homepage is contained within the distfiles url, common for github projects, a cleaner expression that is often used in VOID packages: `distfiles="${homepage}/archive/${version}.tar.gz"`

Reply:
@kawaiiamber this used to be the case, but templates are moving away from this style.

Reply:
Seems reasonable and cleaner for me to use `distfiles="${homepage}/archive/${version}.tar.gz"`

Reply:
see #27735

Reply:
@paper42 it doesnt seem like they reached to a conclusion. now i am confused...

Reply:
Correct, the discussion is currently inconclusive - feel free to use either, just thought I'd mention since I was reviewing, anyway.

Reply:
It's not long enough to require using `homepage`, but it's fine as is now as well.